app/src/Homepage/Homepage.tsx

```diff
@@ -85,10 +85,73 @@ export const LevelTab: React.FC<{ classlevel: string }> = ({ classlevel }) => {
     );
 }
 
+/* 
+  Template for the rating boxes.
+
+  Parameters:
+  label - Text in the center of the box.
+  rating - Determines the color and position of the box 
+           (Difficulty, workload, practicality).
+*/
+const RatingBox: React.FC<{label: string, rating: string}> = ({ label, rating }) => {
+  const dynamicclassname = `ratingbox ratingbox-${rating}`;
+
+  return (
+    <div className={dynamicclassname}>
+      {label}
+    </div>
+  );
+};
+
+type RatingDescProps = {
+  children: React.ReactNode;
+}
+
+/*
+  Template for the descriptions underneath the rating boxes that accepts
+  children as the text for the description.
+*/
+export const RatingDesc: React.FC<RatingDescProps> = ( props: RatingDescProps ) => {
+  
+  return (
+      <div className="ratingdesc">
+          {props.children}
+      </div>
+  );
+}
+
 export const InnerPage: React.FC = () => {
     return (
         <div className="innerpage">
-            {/* This is the content of the inner page */}
+          <h1 className="welcome">
+            Welcome to RateMyCSE
+          </h1>
+          <h3>Wondering if a UW CSE class is as hard as they say?</h3>
+          <div className="howitworks">
+            <h2>How it works:</h2>
+            <h3 className="explanation">
+              Students will post ratings based on 3 categories
+            </h3>
+            <div className="ratingsflexbox">
+              <RatingBox label="Difficulty" rating="diff" />
+              <RatingBox label="Workload" rating="work" />
+              <RatingBox label="Practicality" rating="prac" />
+            </div>
+            <div className="ratingsflexbox">
+              <RatingDesc>
+                A combination of how hard the class material was to 
+                understand and how big of a workload there was
```

Comment:
Nitpick (feel free to ignore for a later discussion), but I don't think one Difficulty should include Workload in its description. Blurs the lines of why Workload is a category more than it needs to.

Reply:
I totally agree! I changed the description, hopefully it's better?